ci-operator/config/opendatahub-io/modelmesh-serving/opendatahub-io-modelmesh-serving-main.yaml

```diff
@@ -15,7 +32,7 @@ releases:
     release:
       architecture: amd64
       channel: stable
-      version: "4.10"
+      version: "4.11"
```

Comment:
Can/should we just be using OpenShift 4.12?

Reply:
yes, agree. I change it to 4.12